test/unit/vl-map-draw-line-action.test.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../../web-component-tester/browser.js"></script>
+
+  <script type="module" src="../../src/vl-map-all.js"></script>
+  <script src="/node_modules/sinon/pkg/sinon.js"></script>
+</head>
+
+<body>
+  <script type="module">
+    import {VlMapDrawLineAction} from '../../src/vl-map-draw-line-action.js';
+
+    suite('vl-map-draw-line-action', () => {
+      setup((done) => {
+        customElements.whenDefined('vl-map-draw-line-action').then(() => done());
+      });
+    
+      test('een lijn teken actie is een map actie', () => {
+        assert.isTrue(VlMapDrawLineAction.isVlMapAction());
+      });
```

Comment:
ook opgemerkt bij punten denk ik, ik zou hier toch de config van uw onderliggende actie proberen te testen, nee? of dat doe je pas als je "default" config echt hebt ...

Reply:
De echte functionaliteit wordt getest op e2e niveau omdat hier op unit niveau eigenlijk alleen maar een constructie gebeurt en daar extra unit testen niet veel gaan bijdragen. Er zitten wel enkele extra unit testen op het abstract niveau.